include/itkScancoImageIO.h

```diff
@@ -88,175 +88,217 @@ class IOScanco_EXPORT ScancoImageIO: public ImageIOBase
    * while others can support 2D, 3D, or even n-D. This method returns
    * true/false as to whether the ImageIO can support the dimension
    * indicated. */
-  virtual bool SupportsDimension(unsigned long dimension) override
```

Comment:
There is both `virtual` and `override` here. One of the previous commits (about override) did not work fully?